src/rollup/index.ts

```diff
@@ -229,7 +229,7 @@ function checkOutputOptions(options: OutputOptions) {
 
 	if (!options.format) {
 		error({
-			message: `You must specify options.format, which can be one of 'amd', 'cjs', 'system', 'es', 'iife' or 'umd'`,
+			message: `You must specify options.format, which can be one of 'amd', 'cjs', 'system', 'esm', 'iife' or 'umd'`,
```

Comment:
If we do this, we should work on a documentation change as well.

Reply:
Although for now, I'm thinking it might just be better to treat `esm` as an alias, with `es` remaining the primary term - moving to consider this being changed in future?